src/commands/network.mjs

```diff
@@ -55,30 +62,25 @@ export class NetworkCommand extends BaseCommand {
     return valuesArg
   }
 
-  prepareValuesFiles (valuesFile) {
-    let valuesArg = ''
-    if (valuesFile) {
-      const valuesFiles = valuesFile.split(',')
-      valuesFiles.forEach(vf => {
-        const vfp = paths.resolve(vf)
-        valuesArg += ` --values ${vfp}`
-      })
-    }
-
-    return valuesArg
-  }
-
-  prepareValuesArg (config = {}) {
+  async prepareValuesArg (config = {}) {
     let valuesArg = ''
     if (config.chartDir) {
       valuesArg = `-f ${config.chartDir}/fullstack-deployment/values.yaml`
     }
 
-    valuesArg += this.prepareValuesFiles(config.valuesFile)
+    if (config.valuesFile) {
+      valuesArg += this.prepareValuesFiles(config.valuesFile)
+    }
+
+    const profileName = this.configManager.getFlag(flags.profileName)
+    const profileValuesFile = await this.profileManager.prepareValuesForFSTChart(profileName)
```

Comment:
```suggestion
    const profileValuesFile = await this.profileManager.prepareValuesForFstChart(profileName)
```
only cap the first letter of an acronym

Reply:
ok, Thanks, I'll make the chages with the rest of the updates.

Reply:
Refactored the method and updated other occurrances.